llpc/util/llpcInternal.cpp

```diff
@@ -84,6 +85,18 @@ CallInst* EmitCall(
     return builder.CreateNamedCall(funcName, pRetTy, args, attribs);
 }
 
+// =====================================================================================================================
+// Emits a amdgcn.reloc.constant intrinsics that represents a relocatable value with the given symbol name.
+llvm::CallInst* EmitRelocationConstant(
+    llvm::LLVMContext* context,                    // The LLVM context
+    llvm::Instruction* pInsertPos,                 // [in] Where to insert this call
```

Comment:
Please change the interface to take an llvm::IRBuilderBase instead. You also don't need to pass the LLVMContext explicitly.

Reply:
Done.